### DISCUSSION TITLE
Renderer checkbox label

### DISCUSSION THREAD

**Discussion by Aetherinox** [markedjs/marked#3270] [Q&A]
Trying this out, and been reading the documentation, however, maybe I've missed something. According to https://marked.js.org/using_pro#renderer; the `checkbox` element only returns the boolean value if the checkbox is checked or not.

I however, do not see where or how the label can be returned from the original markdown so that I can make adjustments slightly to the label styling. 

```
let renderer = new marked.Renderer();
renderer.checkbox = function(bChecked) {
   return `<input class="wrapper-checkbox" type="checkbox">${label}`
};
```

**Comment by UziTech** [comment]
The checkbox renderer only returns the checkbox HTML. The listitem render renders the whole listitem including the checkbox HTML if it has it. https://github.com/markedjs/marked/blob/91ee15b2d43da92f751165c88d1a78ebc3b99114/src/Renderer.ts#L58-L60

**Reply by Aetherinox** [reply]
Ah ok, thanks. Didn't realize check-boxes would also come in as normal lists; Thought they were separate.
